eras/shelley/impl/src/Cardano/Ledger/Shelley/Governance.hs

```diff
@@ -87,6 +89,18 @@ class
   -- | Lens for accessing the previous protocol parameters
   prevPParamsGovStateL :: Lens' (GovState era) (PParams era)
 
+  -- | Getter for accessing the future protocol parameters.
+  --
+  -- This getter is only reliable and efficient 2 stability before the end of the
```

Comment:
"reliable" is maybe too ambiguous here, maybe "stable"?

Reply:
I think it is a perfect wording here precisely because it is ambiguous. It means if you call this function earlier then 2 stability before the end of the epoch, you cannot rely on the answer. It is unreliable in the initial part of the epoch by design, so people don't try to invoke it.

Reply:
FTR: Judging from https://github.com/IntersectMBO/ouroboros-consensus/pull/1078#discussion_r1581565528, the goal is now to make it "reliable" everywhere?

Reply:
Yep, made it reliable everywhere